client/flags.go

```diff
@@ -79,6 +80,7 @@ func PostCommands(cmds ...*cobra.Command) []*cobra.Command {
 		c.Flags().Uint64(FlagSequence, 0, "Sequence number to sign the tx")
 		c.Flags().String(FlagMemo, "", "Memo to send along with transaction")
 		c.Flags().String(FlagFees, "", "Fees to pay along with transaction; eg: 10stake,1atom")
+		c.Flags().String(FlagGasPrices, "", "Gas prices to determine (override) the transaction fee; eg: 1stake")
```

Comment:
Per the below code it doesn't override - `gas-prices` and `fees` are just mutually exclusive.

Reply:
True. Updated.